contracts/src/LightClient.sol

```diff
@@ -84,6 +87,12 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     /// @notice a flag that indicates when a permissioned provrer is needed
     bool public permissionedProverEnabled;
 
+    /// @notice an array to store the L1 Block Heights where the finalizedState was updated
+    uint256[] public l1BlockUpdates;
+
+    /// @notice the threshold used to determine if hotshot is down
+    uint32 public delayThreshold;
```

Comment:
@alysiahuggins @nomaxg @ImJeremyHe @philippecamacho Since rollups may have different thresholds, wouldn't it make more sense to not have this variable in the light client contract? I think this contract should only store the sliding window and add a "getter" function that takes delay threshold as parameter so the rollup can call `isHotShotLive(l1BlockNum, delayThreshold)` with a delay threshold of their choice.

Reply:
@sveitser Yeah that makes sense. I think there are actually 2 questions here:
- Is the hotshot live?
- When should the rollups use the escape hatch?

For the 1st question, I think whether hotshot is live or not should merely determined by us. We are supposed to select this `delayThreshold` according to the properties of HotShot.
For the 2nd question, rollups can have their own choice. But I guess it would also be reasonable that assuming they would use the escape hatch as long as the hotshot is down

Reply:
I agree with @ImJeremyHe's take

Reply:
What's the advantage of having the variable in the contract? I don't think there is any. We want rollups to be able to set the thresholds themselves. Having it in the contract only makes it inflexible (same for all rollups) and we have to make the decision what the value should be. We probably can't make a good decision on the value either because this is not a feature that we are going to use.

Maybe we should use a different name than `isHotShotLive`, but I don't think it matters that much here. We could call it `shouldTriggerEscapeHatch` for example.

cc @nomaxg @jbearer

Reply:
resolving this as we decided to remove it from the contract and allow the user to specific the delayThreshold in their request